conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -165,6 +169,31 @@ TEST_F(
   }
 }
 
+#ifdef USE_ZESINIT
+TEST_F(
+    SYSMAN_DEVICE_TEST,
+    GivenValidDeviceWhenRetrievingSubDevicesThenEnsureNoSubDeviceUUIDMatchesDeviceUUID) {
```

Comment:
Looking at the https://jira.devtools.intel.com/browse/VLCLJ-2248, the objective is to validate zesDeviceGetSubDeviceProperties.
So may be a test like "ValidateZesDeviceGetSubDevicePropertiesReturnsCorrectProperties" is required

Reply:
The Jira: VLCLJ-2251 validates whether each sub-device is returning the correct properties. The Jira: VLCLJ-2248 focuses on validating the sub-device UUIDs. Thank you.

Reply:
Oh .. Ok.